ciao-launcher/docker.go

```diff
@@ -214,18 +274,26 @@ func (d *docker) createImage(bridge string, userData, metaData []byte) error {
 		return err
 	}
 
-	config, hostConfig, networkConfig := d.createConfigs(bridge, userData, metaData)
+	volumes, err := d.mountVolumes()
+	if err != nil {
+		glog.Errorf("Unable to mount container volumes %v", err)
```

Comment:
If you have multiple volumes to mount and the nth one fails. Then you'll still have the n-1 others still mounted. Do they need to be cleaned up?